index.js

```diff
@@ -15,7 +15,8 @@ function detectRepoTestFiles(dir, options) {
 
     return globby([
         '**/{test,spec}?(s)/**/*',            // Conventional test dirs (including deep)
-        '{test,spec}?(s).*',                  // Conventional root test files
+        '**/*[.,-]{test,spec}?(s).*',         // Suffix-style test files (foo-test.js, foo.test.js)
```

Comment:
Ah, yep you're right... Looks like I hybridized the `{}` and `[]` patterns 😝 

Adjusting now....